src/pymatgen/io/vasp/outputs.py

```diff
@@ -800,10 +800,10 @@ def run_type(self) -> str:
             "--",
             "None",
         }:
-            incar_tag = self.incar.get("METAGGA", "").strip().upper()
```

Comment:
I believe this would be covered by https://github.com/materialsproject/pymatgen/blob/6992aeef9847b03da81c80dabe27c84010e28abc/src/pymatgen/io/vasp/inputs.py#L1029